### PR TITLE
IBX-4168: Unified search reference

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket | (https://issues.ibexa.co/browse/IBX-4168)
| Versions | 4.4+

Unified search reference: pages in TOC reorganized, content updated, information verified.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Redirects cover removed/moved pages
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/snippets/aggregation_arguments.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+- `name` - name of the Aggregation object
```
Comment (adamwojs) [inline docs/snippets/aggregation_arguments.md:1]:
```suggestion
- `name` - name of the Aggregation
```

You can use simply `name of the aggregation` or `name of the aggregation definition`

File: docs/snippets/aggregation_arguments.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+- `contentTypeIdentifier` - string representing the Content Type identifier
+- `fieldDefinitionIdentifier` - string representing the Field identifier
```
Comment (adamwojs) [inline docs/snippets/aggregation_arguments.md:3]:
```suggestion
- `fieldDefinitionIdentifier` - string representing the Field Definition identifier
```

File: docs/search/criteria_reference/customprice_criterion.md [hunk 3]
```diff
@@ -4,10 +4,10 @@ The `CustomPrice` Search Criterion searches for products by their custom price f
...
 ## Arguments
 
-- `value` - a `Money` object representing the price in a specific currency.
-- (optional) `operator` - Operator constant (EQ, GT, GTE, LT, LTE, default EQ).
-- (optional) `customerGroup` - a `CustomerGroupInterface` object representing the customer group to prices prices for.
-If you do not provide a customer group, the query uses the group related to the current user.
+- `value` - a `Money` object representing the price in a specific currency
+- (optional) `operator` - Operator constant (EQ, GT, GTE, LT, LTE, default EQ)
+- (optional) `customerGroup` - a `CustomerGroupInterface` object representing the customer group to prices prices for
+If you do not provide a customer group, the query uses the group related to the current user
```
Comment (MagdalenaZuba) [inline docs/search/criteria_reference/customprice_criterion.md:10]:
```suggestion
If you do not provide a customer group, the query uses the group related to the current user.
```

File: docs/search/criteria_reference/customprice_criterion.md [hunk 4]
```diff
@@ -4,10 +4,10 @@ The `CustomPrice` Search Criterion searches for products by their custom price f
 
 ## Arguments
 
-- `value` - a `Money` object representing the price in a specific currency.
-- (optional) `operator` - Operator constant (EQ, GT, GTE, LT, LTE, default EQ).
-- (optional) `customerGroup` - a `CustomerGroupInterface` object representing the customer group to prices prices for.
-If you do not provide a customer group, the query uses the group related to the current user.
+- `value` - a `Money` object representing the price in a specific currency
+- (optional) `operator` - Operator constant (EQ, GT, GTE, LT, LTE, default EQ)
+- (optional) `customerGroup` - a `CustomerGroupInterface` object representing the customer group to prices prices for
```
Comment (MagdalenaZuba) [inline docs/search/criteria_reference/customprice_criterion.md:9]:
```suggestion
- (optional) `customerGroup` - a `CustomerGroupInterface` object representing the customer group to show prices for.
```

File: docs/search/criteria_reference/product_search_criteria.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+|-----|-----|
+|[CheckboxAttribute](checkboxattribute_criterion.md)|Value of product's checkbox attribute|
+|[ColorAttribute](colorattribute_criterion.md)|Value of product's color attribute|
+|[CreatedAt](createdat_criterion.md)|Date and time when product was created|
+|[CreatedAtRange](createdatrange_criterion.md)|Date and time range when product was created|
+|[FloatAttribute](floatattribute_criterion.md)|Value of product's float attribute|
+|[IntegerAttribute](integerattribute_criterion.md)|Value of product's integer attribute|
+|[SelectionAttribute](selectionattribute_criterion.md)|Value of product's selection attribute|
+|[ProductAvailability](productavailability_criterion.md)|Product's availability|
+|[ProductCategory](productcategory_criterion.md)|Product category assigned to product|P
```
Comment (MagdalenaZuba) [inline docs/search/criteria_reference/product_search_criteria.md:23]:
```suggestion
|[ProductCategory](productcategory_criterion.md)|Product category assigned to product|
```

File: docs/search/criteria_reference/simplemeasurementattribute_criterion.md [hunk 6]
```diff
@@ -4,8 +4,8 @@ The `SimpleMeasurementAttribute` Search Criterion searches for products by the v
 
 ## Arguments
 
--  `identifier` - string representing the attribute.
--  `value` - `\Ibexa\Contracts\Measurement\Value\SimpleValueInterface` object representing the attribute value.
+- `identifier` - string representing the attribute
+- `value` - `\Ibexa\Contracts\Measurement\Value\SimpleValueInterface` object representing the attribute value
```
Comment (MagdalenaZuba) [inline docs/search/criteria_reference/simplemeasurementattribute_criterion.md:8]:
```suggestion
- `value` - `Ibexa\Contracts\Measurement\Value\SimpleValueInterface` object representing the attribute value
```

File: docs/search/sort_clause_reference/customprice_sort_clause.md [hunk 7]
```diff
@@ -4,10 +4,10 @@ The `CustomPrice` Sort Clause sorts search results by the product's custom price
...
 ## Arguments
 
-- `currency` - a `CurrencyInterface` object representing the currency to check price for.
-- `sortDirection` (optional) - Query or LocationQuery constant, either `Query::SORT_ASC` or `Query::SORT_DESC`.
-- `customerGroup` (optional) - a `CustomerGroupInterface` object representing the customer group to check prices for.
-If you do not provide a customer group, the query uses the group related to the current user.
+- `currency` - a `CurrencyInterface` object representing the currency to check price for
+[[= include_file('docs/snippets/sort_direction.md') =]]
+- (optional) `customerGroup` - a `CustomerGroupInterface` object representing the customer group to check prices for
+If you do not provide a customer group, the query uses the group related to the current user
```
Comment (MagdalenaZuba) [inline docs/search/sort_clause_reference/customprice_sort_clause.md:10]:
```suggestion
- (optional) `customerGroup` - a `CustomerGroupInterface` object representing the customer group to check prices for.
If you do not provide a customer group, the query uses the group related to the current user.
```

File: docs/search/criteria_reference/search_criteria_reference.md [hunk 8]
```diff
@@ -21,73 +21,54 @@ Due to this storage limitation, searching content using the Country Field Type o
...
+|[ContentTypeGroupId](contenttypegroupid_criterion.md)|ID of the Content item's Content Type group|&#10004; |&#10004; |&#10004; |
+|[ContentTypeId](contenttypeid_criterion.md)|ID of the Content item's Content Type|&#10004; |&#10004; |&#10004; |
+|[ContentTypeIdentifier](contenttypeidentifier_criterion.md)|Identifier of the Content item's Content Type|&#10004; |&#10004; |&#10004; |
+|[CurrencyCodeCriterion](currencycode_criterion.md)|Currency code|&#10004; |&#10004; |&#10004; |
+|[DateMetadata](datemetadata_criterion.md)|The date when content was created or last modified|&#10004; |&#10004; |&#10004; |
+|[Depth](depth_criterion.md)|Location depth in the Content tree| |&#10004; |&#10004; |
+|[Field](field_criterion.md)|Content of one of Content item's Fields|&#10004; |&#10004; | |
+|[FieldRelation](fieldrelation_criterion.md)|Content items the content in question has Relations to|&#10004; |&#10004; | |
+|[FullText](fulltext_criterion.md)|Full text content of a Content item's Fields|&#10004; |&#10004; | |
+|[IsCurrencyEnabledCriterion](iscurrencyenabled_criterion.md)|Whethe a specificed currency is enabled in the system| | | |
```
Comment (MagdalenaZuba) [inline docs/search/criteria_reference/search_criteria_reference.md:37]:
```suggestion
|[IsCurrencyEnabledCriterion](iscurrencyenabled_criterion.md)|Whether a specified currency is enabled in the system| | | |
```

**Review by MagdalenaZuba — CHANGES_REQUESTED** [review]

**Review by MagdalenaZuba — APPROVED** [review]

**Review by adamwojs — APPROVED** [review]

**Pull request merged**